app/assets/javascripts/common/admins.js

```diff
@@ -2,13 +2,14 @@
 // elements
 //
 const ACCESS_LIST_INPUT_SELECTOR = "input.access-input"
+const ACCESS_LEVEL_ID = "access-level"
```

Comment:
Why is this indirection necessary?

Reply:
Because it's being used in multiple places, once via JQuery (because of the selectpicker library) and once via JS. Since JS is very forgiving about errors, it's best to use a constant for these things to avoid regressions.